Tests/UnitTests/Purchasing/PurchasesTests.swift

```diff
@@ -2182,42 +2182,6 @@ class PurchasesTests: XCTestCase {
         expect(invokedMethodParams.appUserID) == identityManager.currentAppUserID
     }
 
-    func testAttributionDataDontSendNetworkAppUserIdIfNotProvided() throws {
```

Comment:
Do these need to be removed now?

Reply:
i thought i was on the adservices branch 🤦‍♀️ reverted